log/README.md

```diff
@@ -1,54 +1,45 @@
+
+
 # package log
 
-`package log` provides a minimal interface for structured logging in services.
-It may be wrapped to encode conventions, enforce type-safety, etc.
-It can be used for both typical application log events, and log-structured data streams.
+    import "/Users/harlow/code/go/src/github.com/go-kit/kit/log"
 
 ## Rationale
 
-TODO
+Package log provides basic interfaces for structured logging.
+
+The fundamental interface is Logger. Loggers create log events from
+key/value data.
```

Comment:
The doc comment as it stands isn't rationale, rather this is overview. But we could expand the it to cover both. Can you please backport the previous overview to the doc comment, and put a big "TODO rationale" at the end?